clients/admin-ui/src/features/common/Layout.tsx

```diff
@@ -39,7 +40,7 @@ const Layout = ({
     skip,
   });
 
-  const showNotificationBanner =
+  const showConfigurationBanner =
```

Comment:
I changed this name because I also named my component `NotificationBanner` but there's a "ConfigurationNotificationBanner" here. IMO there's too much config notification logic in this component (which should be a very general Layout component) and it should be refactored out of here if possible.

made an issue: https://github.com/ethyca/fides/issues/2843